content/yaml-code-signing/signing-macos.md

```diff
@@ -70,11 +61,11 @@ environment:
 
 - `APP_STORE_CONNECT_PRIVATE_KEY`
 
-  This is the private API key downloaded from App Store Connect. You'll need to [encrypt](../building/encrypting/) the **contents** of the file in the codemagic.yaml editor UI (and not the file itself). On macOS, you can use `pbcopy < AuthKey_XXXXXX.p8` to copy the contents of the private key and paste this into the UI encryption tool.
+  This is the private API key downloaded from App Store Connect. On macOS, you can use `pbcopy < AuthKey_XXXXXX.p8` to copy.
```

Comment:
is there a part of instructions missing here?

Reply:
no, I think it getting repeated over and over again to secure the variables. It is already mentioned above.